recipe_scrapers/mob.py

```diff
@@ -37,6 +37,9 @@ def yields(self):
     def image(self):
         return self.recipe_json.get("image", [])[0]["url"]
 
+    def nutrients(self):
+        return {"servingSize": f"{self.recipe_json.get('servingSize')}"}
+
```

Comment:
Could/should we drop this `nutrients` implementation @jacksgreen?  (I'm guessing it may have been experimental or a work-in-progress?  I think the `yields` method provides the serving size info already.

Reply:
right, removed 👍